TestProject/output/BaseOutput.php

```diff
@@ -0,0 +1,52 @@
+<?php
+namespace output;
+
+use options\Getopt;
+use cellularAutomat\Board;
+
+/**
+ * BaseOutput class which provides the functions for pluggable-output.
+ *
+ * @class BasOutput
```

Comment:
Please remove that line, it is not necessary, everyone knows that this is a class (even PHPStorm or a documentation tool)